.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Token Imbalances",
+  "build": {
+    "dockerfile": "../Dockerfile",
+    "context": ".."
+  },
+  "postCreateCommand": "python -m pip install -r requirements.txt"
```

Comment:
Not sure if this works for you, but in the other project I set this up for recently I had to add

```
"mounts": [
		"source=${localEnv:HOME}/.ssh,target=/root/.ssh,type=bind,consistency=cached"
	],
```

to get `git pull` commands to work and

```
  "customizations": {
        "vscode": {
            "extensions": [
                "ms-python.python"
            ],
            "settings": {
                "python.testing.pytestArgs": [
                    "."
                ],
                "python.testing.unittestEnabled": false,
                "python.testing.pytestEnabled": true,
                "python.formatting.provider": "black",
                "python.linting.mypyEnabled": true,
                "python.linting.enabled": true
            }
        }
    }
```

to have language plugin support.